SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -296,7 +325,6 @@
                 "ctags",
                 "curl",
                 "cyrus-sasl",
-                "Cython",
```

Comment:
Don't remove cython here since you're not modifying it.

Reply:
Done